### PR TITLE
ocp4: Add documentation for e2e tests

### PR DESCRIPTION
**Pull request by JAORMX**

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now ef57bba9c7b250fe31b5dcd901a1c86b9fae44d3

File: docs/manual/developer_guide.adoc [hunk 1]
```diff
@@ -1925,6 +1925,172 @@ Tests are added using the add_test cmake call. Each test should finish with a 0
...
+which is a successful status.
+
+The e2e test run will time out at *15 minuntes* if a script doesn't converge.
+
+Note that the scripts will be run in parallel, but the test run will wait for
+all of them to be done.
+
+===== Running the e2e tests on a local cluster
+
+Note that it's possible to run the e2e tests on a cluster of your choice.
```
Comment (jhrozek) [inline docs/manual/developer_guide.adoc:2068]: Should we note that there's no cleanup yet?

Comment (JAORMX) [reply docs/manual/developer_guide.adoc:2068]: That's a good point. I'll add that here.

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now d4cadb770eb5229bd3fb3e5151db44a6a9fcc67c

**Review by jhrozek — APPROVED** [review]
Thanks, this looks good

**Pull request merged**